npm-shrinkwrap.json

```diff
@@ -2833,6 +2833,14 @@
         "@types/node": "*"
       }
     },
+    "@types/rsync": {
```

Comment:
I think these new `@types/*` dependencies (coming from `balena-sync`) should not have found their way into the `balena-cli` `npm-shrinkwrap.json` file, because `balena-cli` does not need them in either dev or prod installs of the balena CLI (e.g. the CLI itself does not import package `'rsync'`, and therefore does not need to install `@types/rsync`). Added a comment to the `balena-sync` PR: https://github.com/balena-io-modules/balena-sync/pull/167/files#r768840867

I suggest a patch PR to `balena-io-modules/balena-sync` to change those dependencies from prod to dev, then amending `package.json` in this PR.

Reply:
It is not a big deal though, and we could merge this PR as it is. However, I just learned that we need to release CLI v13 in the next few days ([FD thread](https://www.flowdock.com/app/rulemotion/i-cli/threads/hdlxb8LTwYxQ8os3tFciOvh0CwD)), and I was reasoning that it might now be best for us to avoid making any non-essential changes to CLI v12, because it would be trickier to apply fixes to CLI v12 after CLI v13 was out (better safe than sorry). Which is to say, I would suggest waiting for CLI v13 to be out (next week) before merging this PR.